sqflite/lib/src/adapter.dart

```diff
@@ -6,133 +6,33 @@ library jaguar_query_sqflite.src;
 
-  sqf.Database get connection => _connection;
-
-  /// Finds one record in the table
-  Future<Map> findOne(Find st) async {
-    String stStr = composeFind(st);
-    List<Map<String, dynamic>> list = await connection.rawQuery(stStr);
-
-    if (list.length == 0) return null;
-
-    return list.first;
-  }
-
-  // Finds many records in the table
-  Future<List<Map>> find(Find st) async {
-    String stStr = composeFind(st);
-    return connection.rawQuery(stStr);
-  }
-
-  /// Inserts a record into the table
-  Future<T> insert<T>(Insert st) async {
-    String strSt = composeInsert(st);
-    return connection.rawInsert(strSt) as Future<T>;
-  }
-
-  /// Inserts or update a record into the table
-  Future<T> upsert<T>(Upsert st) async {
-    String strSt = composeUpsert(st);
-    return connection.rawInsert(strSt) as Future<T>;
-  }
-
-  /// Inserts or update records into the table
-  Future<void> upsertMany<T>(UpsertMany st) async {
-    List<String> strSt = composeUpsertMany(st);
-    final batch = connection.batch();
-    for (var query in strSt) {
-      batch.execute(query);
-    }
-    return batch.commit(noResult: true);
-  }
-
-  /// Inserts many records into the table
-  Future<void> insertMany<T>(InsertMany st) {
-    String strSt = composeInsertMany(st);
-    return connection.execute(strSt);
-  }
-
-  /// Updates a record in the table
-  Future<int> update(Update st) {
-    String strSt = composeUpdate(st);
-    return connection.rawUpdate(strSt);
-  }
-
-  /// Updates a record in the table
-  Future<void> updateMany(UpdateMany st) {
-    List<String> strSt = composeUpdateMany(st);
-    final batch = connection.batch();
-    for (var query in strSt) {
-      batch.execute(query);
-    }
-    return batch.commit(noResult: true);
-  }
-
-  /// Deletes a record from the table
-  Future<int> remove(Remove st) {
-    String strSt = composeRemove(st);
-    return connection.rawDelete(strSt);
-  }
-
-  /// Creates the table
-  Future<void> createTable(Create statement) async {
-    String strSt = composeCreate(statement);
-    await connection.execute(strSt);
-  }
-
-  /// Create the database
-  Future<void> createDatabase(CreateDb st) async {
-    String strSt = composeCreateDb(st);
-    await connection.execute(strSt);
-  }
-
-  /// Drops tables from database
-  Future<void> dropTable(Drop st) async {
-    String strSt = composeDrop(st);
-    await connection.execute(strSt);
-  }
-
-  Future<void> dropDb(DropDb st) async {
-    String strSt = composeDropDb(st);
-    await connection.execute(strSt);
-  }
+  sqf.Database get connection => _connection.connection;
 
   @override
-  Future<void> alter(Alter st) async {
-    String strSt = composeAlter(st);
-    await connection.execute(strSt);
+  Future<T> run<T>(Future<T> Function(Connection<sqf.Database> conn) task,
+      {Connection<sqf.Database> withConn}) async {
+    return await super.run(task, withConn: _connection);
   }
 
   T parseValue<T>(dynamic v) {
```

Comment:
not used anymore, please remove it, as it's under connection and duplicate

Reply:
done